calculator/calculator.py

```diff
@@ -0,0 +1,218 @@
+"""Calculator module.
+
+This module allows evaluation of strings which represent mathematical
+expressions. The supported operations are declared in operators dictionary.
+
+Example:
+    To use this module, import the evaluate() function:
+
+        from calculator import evaluate
+
+    The parameter of this function is a mathematical expression (str). For
+    example, "1 + (3^-2) * (4! - 5$ 2)".
```

Comment:
If you provide an example, just write how the code is suppose to run.
There are too much words here, the evaluate function explanation should be written under its docs.

```suggestion
    from calculator import evaluate
    evaluate("1+1")
```